File: all/abc268/abc268_b/abc268_b.py
s = input()
t = input()
if len(s) > len(t): print("No")
else:
  ans = True
  cnt = 0
  while cnt < len(s):
    if s[cnt] != t[cnt]: ans = False
    cnt += 1
  print("Yes" if ans else "No")

File: abc/abc268/abc268_b/abc268_b.cs
using System;

class A
{
  static void Main()
  {
    string s = Console.ReadLine();
    string t = Console.ReadLine();

    bool ans = true;
    if(s.Length > t.Length)
    {
      ans = false;
    }

    for(int i = 0; i < s.Length; i++)
    {
      if(i >= t.Length)
      {
        break;
      }

      if(s[i] != t[i])
      {
        ans = false;
        break;
      }
    }

    if(ans)
    {
      Console.WriteLine("Yes");
    }
    else
    {
      Console.WriteLine("No");
    }
  }
}
